src/utils/utils.ts

```diff
@@ -1,3 +1,10 @@
 export function format(first: string, middle: string, last: string): string {
   return (first || '') + (middle ? ` ${middle}` : '') + (last ? ` ${last}` : '');
 }
+
+export function getSlottedNodes(root: HTMLElement, nodeName: string): Array<Node> {
+  return root.shadowRoot
+    .querySelectorAll('slot')[0]
```

Comment:
1) Can this be `querySelector` to return the first node?
2) What happens if there are none?

Reply:
If there are no `<slot>`s, then this function is being used somewhere that it shouldn't be. I'll leave a comment on it, but it is explicitly meant to be used to return the nodes inside of a `<slot>` matching `nodeName`, all inside a given root node.